Services/CO.CDP.Person.WebApi.Tests/UseCase/ClaimPersonInviteUseCaseTest.cs

```diff
@@ -40,7 +41,7 @@ public async Task Execute_Throws_UnknownPersonException_When_Person_Not_Found()
             .ReturnsAsync((OrganisationInformation.Persistence.Person)null!);
 
         var exception = await Assert.ThrowsAsync<UnknownPersonException>(() => _useCase.Execute(command));
-        Assert.Equal($"Unknown person {command.personId}.", exception.Message);
+        exception.Message.Should().Match($"Unknown person {command.personId}.");
```

Comment:
For simple string equal assertion, use `.Be({expected_value})` 
https://fluentassertions.com/basicassertions/